CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ Note: this changelog is for the shields.io server. The changelog for the badge-m
 
 ---
 
+## server-2021-10-01
+
+- Add [ClearlyDefined] service [#6944](https://github.com/badges/shields/issues/6944)
+- Update Shields production access doc [#7079](https://github.com/badges/shields/issues/7079)
+- refactor: cleanup AppveyorTests with by using shared helpers [#7077](https://github.com/badges/shields/issues/7077)
```

Comment:
Looking back at the notes for prior release, I gather these are the types of things we're removing manually to clean up the notes?

Reply:
Yeah I generally remove anything to do with docs, internals etc